
- A simple SNP calling pipaline
https://www.ebi.ac.uk/sites/ebi.ac.uk/files/content.ebi.ac.uk/materials/2014/140217_AgriOmics/dan_bolser_snp_calling.pdf

- Calling variants with GATK and with samtools  
https://angus.readthedocs.io/en/2013/snp_tutorial.html

- Variant Calling (SNPs/INDELs)  
https://ucdavis-bioinformatics-training.github.io/2019-Alliance-for-Global-Health-and-Science-Makerere-University_Variants/variant_analysis/variant_calling.html

- variant detection on exeom data  
https://www.sciencedirect.com/science/article/pii/S2666166722002982

- The evaluation of Bcftools mpileup and GATK HaplotypeCaller for variant calling in non-human species  
https://www.nature.com/articles/s41598-022-15563-2

- Summoning insights: NGS variant calling best practices  
https://www.ogt.com/ca/about-us/ogt-blog/summoning-insights-ngs-variant-calling-best-practices/

- Sequencing depth and coverage: key considerations in genomic analyses  
https://www.nature.com/articles/nrg3642

- How to calculate the coverage for a NGS experiment  
https://www.ecseq.com/support/ngs/how-to-calculate-the-coverage-for-a-sequencing-experiment

