
- A simple SNP calling pipaline
https://www.ebi.ac.uk/sites/ebi.ac.uk/files/content.ebi.ac.uk/materials/2014/140217_AgriOmics/dan_bolser_snp_calling.pdf

- A_beginners_guide_to_Call_SNPs_and_indels_Part_II  
https://yulijia.net/slides/bioinfomatcis_for_medical_students/2019-07-31-A_beginners_guide_to_Call_SNPs_and_indels_Part_II.html

- Introduction to Variant Discovery with GATK Best Practices  
https://qcb.ucla.edu/wp-content/uploads/sites/14/2016/03/GATKwr12-1-GATK_primer.pdf

- Genotype likelihood  
https://gatk.broadinstitute.org/hc/en-us/articles/360035890731-Likelihoods-and-Probabilities


- formation_variants_detection_2017  
https://wikis.univ-lille.fr/bilille/_media/formation_variants_detection_2017.pdf

- Variant Calling (SNPs/INDELs)  
https://ucdavis-bioinformatics-training.github.io/2019-Alliance-for-Global-Health-and-Science-Makerere-University_Variants/variant_analysis/variant_calling.html

- Calling variants with GATK and with samtools  
https://angus.readthedocs.io/en/2013/snp_tutorial.html

- Variant calling and filtering for INDELs  
https://genome.sph.umich.edu/w/images/b/b4/Variant_Calling_and_Filtering_for_INDELs.pdf

- Indel-based	Realignment  
https://qcb.ucla.edu/wp-content/uploads/sites/14/2016/03/GATKwr12-3-IndelRealignment.pdf

- variant detection on exome data  
https://www.sciencedirect.com/science/article/pii/S2666166722002982

- The evaluation of Bcftools mpileup and GATK HaplotypeCaller for variant calling in non-human species  
https://www.nature.com/articles/s41598-022-15563-2

- Summoning insights: NGS variant calling best practices  
https://www.ogt.com/ca/about-us/ogt-blog/summoning-insights-ngs-variant-calling-best-practices/

- Sequencing depth and coverage: key considerations in genomic analyses  
https://www.nature.com/articles/nrg3642

- How to calculate the coverage for a NGS experiment  
https://www.ecseq.com/support/ngs/how-to-calculate-the-coverage-for-a-sequencing-experiment

- Removing reference bias and improving indel calling in ancient DNA data analysis by mapping to a sequence variation graph  
https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02160-7

- spurious variants caused by alignment artifacts around indels  
https://github.com/andersen-lab/ivar/issues/85

- Accurate genotyping across variant classes and lengths using variant graphs  
https://www.nature.com/articles/s41588-018-0145-5

- Genotype and SNP calling from next-generation sequencing data  
https://www.nature.com/articles/nrg2986

Nucleotide diversity  
https://en.wikipedia.org/wiki/Zygosity#Heterozygosity_in_population_genetics

https://en.wikipedia.org/wiki/Nucleotide_diversity

#### Articles  propos√©s 

Insights into human genetic variation and population history from 929 diverse genomes  
https://www.science.org/doi/10.1126/science.aay5012  

Fast and accurate metagenotyping of the human gut microbiome with GT-Pro  
https://www.nature.com/articles/s41587-021-01102-3

github: https://github.com/zjshi/gt-pro  

Genomic Analysis of European Drosophila melanogaster Populations Reveals Longitudinal Structure, Continent-Wide Selection, and Previously Unknown DNA Viruses  
https://academic.oup.com/mbe/article/37/9/2661/5837682


